app/views/admin/overview/index.html.erb

```diff
@@ -1,104 +1,211 @@
-              <%= "Open report".pluralize(@open_abuse_reports_count) %>
-            </a>
-          </p>
+</h1>
+
+<div class="flex flex-col gap-4">
+  <div class="flex flex-col m:flex-row gap-4">
+    <section class="crayons-card p-7 m:basis-2-3">
+      <header class="flex justify-between items-center mb-7">
+        <h2 class="crayons-subtitle-1">Analytics and trends</h2>
+        <div class="relative w-auto">
+          <button type="button" class="c-btn c-btn--icon-alone dropdown-trigger whitespace-nowrap" id="timeperiods-trigger" aria-haspopup="true" aria-expanded="false" aria-controls="options-dropdown">
+            <%= crayons_icon_tag("calendar", title: "Timeframes", class: "c-btn__icon") %>
+          </button>
+          <form action="" id="timeperiods-dropdown" class="crayons-dropdown right-0 left-0 s:left-auto">
+            <fieldset class="mb-2">
+              <ul class="mb-0 p-2 flex flex-col gap-3">
+                <li class="crayons-field crayons-field--radio">
+                  <input type="radio" name="timeperiod" value="7" id="7days" class="crayons-radio" checked />
+                  <label for="7days" class="crayons-field__label">Last 7 days</label>
+                </li>
+                <li class="crayons-field crayons-field--radio">
+                  <input type="radio" name="timeperiod" value="14" id="14days" class="crayons-radio" />
+                  <label for="14days" class="crayons-field__label">Last 14 days</label>
+                </li>
+                <li class="crayons-field crayons-field--radio">
+                  <input type="radio" name="timeperiod" value="30" id="30days" class="crayons-radio" />
+                  <label for="30days" class="crayons-field__label">Last 30 days</label>
+                </li>
+              </ul>
+            </fieldset>
+            <button type="submit" class="c-btn c-btn--secondary w-100">Submit</button>
+          </form>
         </div>
-      </section>
-    </article>
-    <h2 class="crayons-subtitle-1 mb-3">Resources</h2>
-    <article class="grid s:grid-cols-2 gap-3" data-controller="ahoy">
-      <section class="crayons-card">
-        <div class="crayons-card__header">
-          <h3 class="crayons-subtitle-2">Forem Admin Guide</h3>
+      </header>
+
+      <div class="grid s:grid-cols-2 gap-7">
+        <div>
+          <header class="color-secondary">
+            <h3 class="fs-l fw-normal"><span class="color-primary fw-bold">18</span> posts</h3>
+            <p class="flex items-center fs-s mb-1">
+              <%= crayons_icon_tag(:increase, width: 18, height: 18, class: "color-accent-success") %>
+              17% increase
+            </p>
+          </header>
+          <canvas
+            id="postsChart"
+            width="400"
+            height="100"
+            class="-mx-1 js-chart"
+            data-days="Mar 21,Mar 22,Mar 23,Mar 24,Mar 25,Mar 26,Mar 27"
+            data-values="2,5,4,7,6,8,9"></canvas>
```

Comment:
These are the two data attributes that generate the graph. So whatever is there (comma-separated) will be used to generate graph.